less/index.less

```diff
@@ -47,13 +49,208 @@ table {
 	border-collapse: collapse;
 	border-spacing: 0;
 }
+html {
+	font-size: 62.5%;
+}
 
 /* Set every element's box-sizing to border-box */
 * {
     box-sizing: border-box;
 }
 
 html, body {
-    height: 100%;
+	height: 100%;
+
+}
+
+@italics: italic;
+@headers: 2rem;
+@paragraphfont:1.9rem;
+@headercolor: pink;
+
+.container {
+	// border: 1px solid rgb(40, 176, 197);
+	max-width: 100%px;
+	margin: 0 auto; 
+	display: flex;
+	flex-wrap: wrap; 
+	flex-direction: column;
+	justify-content: center;
+	align-items: center;
+}
+
+header {
+	// border: 1px solid blue;
+	width: 100%;
+	display: flex;
+	flex-wrap: wrap;
+}
+
+header nav {
+	// border: 1px solid green;
+	display: flex;
+	flex-wrap: wrap;
+	justify-content: space-around;
+	margin-top: 30px;
+	width: 100%;
+}
+
+header nav a {
+	// border: 1px solid pink;
+	color: pink;
+	text-decoration: none;
+	font-size: 2rem;
+}
+
+.welcome {
+	// border: 1px dotted orange;
+	width:60%;	
+	margin-bottom: 20px;
+	border-bottom: 2px solid black;
+	h1 {
+		font-size: @headers;
+		// border: 1px solid red;
+		font-style: @italics;
+		display: flex;
+		justify-content: center;
+		font-size: 4rem;
+		flex-wrap: wrap;
+		align-items: center;
+	}
+	#image-one {
+		// width: 370px;
+		height: 235px;
+		margin-top: 10px;
+		margin-bottom: 10px;
```

Comment:
Perhaps get rid of the margin-top and margin-bottom. Instead, let's combine them into one property:
```
margin: 30px 0px;
```

This will add 30px's of margin to the top and bottom, while making no amendments horizontally.  I think the added px's will also give your image some room to breathe.